content/guides/tasks/assignments/index.md

```diff
@@ -0,0 +1,11 @@
+---
+rank: 1
+related_endpoints: []
+related_guides:
+required_guides: []
+alias_paths: []
+---
+
+# Assigning tasks
+
+When a task is delegated to a user in Box, that creates a task assignment.
```

Comment:
Is this missing some content?

Reply:
Since it was an index page, I kept it pretty light. I can definitely flesh it out a bit more or include links as necessary.

Reply:
Ah my bad. Yes, index page